### ISSUE TITLE
Please add support for Angular 2 RC 6 NgModule

### ISSUE THREAD

**Issue by naveedahmed1** [cstefanache/angular2-img-cropper#55]
Though the cropper works with RC 6, but it would be better to add support for Angular 2 RC 6 NgModule.

**Comment by rubimazaki** [comment]
Hey does it have anything to do with Ionic2/Angular2 compiler emitting:
```
export * from "./src/imageCropperComponent";
^
ParseError: 'import' and 'export' may appear only with 'sourceType: module'

```

**Comment by KingsDevelopment** [comment]
Fixed by #62
